src/constants/commonResponses.ts

```diff
@@ -0,0 +1,40 @@
+import { CACHE_HEADERS } from './cache';
+
+export const METHOD_NOT_ALLOWED = new Response(undefined, {
+  status: 405,
+  headers: {
+    allow: 'GET, HEAD, OPTIONS',
+    'cache-control': CACHE_HEADERS.failure,
+  },
+});
+
+export const BAD_REQUEST = new Response(undefined, {
+  status: 400,
+  headers: {
+    'cache-control': CACHE_HEADERS.failure,
+  },
+});
+
+export const FILE_NOT_FOUND = (request: Request): Response => {
```

Comment:
I don't think explicit types are needed here, TS will already infer this is of type Response

Reply:
Maybe, but eslint requires them under `@typescript-eslint/explicit-function-return-type`